### PR TITLE
7.x-BPL

### PR DESCRIPTION
**Pull request by MorganDawe**

### CODE REVIEW

File: platform_main_feature.features.menu_links.inc [hunk 1]
```diff
@@ -147,44 +198,64 @@ function platform_main_feature_menu_default_menu_links() {
     'weight' => -49,
     'customized' => 1,
   );
-  // Exported menu link: menu-header-menu_about-this-site:uuid/node/3dfbfeaf-cb77-4ac0-b16f-145af17649b5.
-  $menu_links['menu-header-menu_about-this-site:uuid/node/3dfbfeaf-cb77-4ac0-b16f-145af17649b5'] = array(
+  // Exported menu link: menu-header-menu_about-us:node/5.
+  $menu_links['menu-header-menu_about-us:node/5'] = array(
     'menu_name' => 'menu-header-menu',
-    'link_path' => 'uuid/node/3dfbfeaf-cb77-4ac0-b16f-145af17649b5',
```
Comment (jordandukart) [inline platform_main_feature.features.menu_links.inc]: No longer uuided?

File: platform_main_feature.features.menu_links.inc [hunk 2]
```diff
@@ -127,6 +161,23 @@ function platform_main_feature_menu_default_menu_links() {
     'weight' => -48,
     'customized' => 1,
   );
+  // Exported menu link: menu-footer-menu_researchers:node/2.
+  $menu_links['menu-footer-menu_researchers:node/2'] = array(
```
Comment (jordandukart) [inline platform_main_feature.features.menu_links.inc]: No uuids?

File: platform_main_feature.features.menu_links.inc [hunk 3]
```diff
@@ -107,6 +124,23 @@ function platform_main_feature_menu_default_menu_links() {
     'weight' => -47,
     'customized' => 1,
   );
+  // Exported menu link: menu-footer-menu_policies:node/10.
+  $menu_links['menu-footer-menu_policies:node/10'] = array(
```
Comment (jordandukart) [inline platform_main_feature.features.menu_links.inc]: UUID.

File: platform_main_feature.features.menu_links.inc [hunk 4]
```diff
@@ -147,44 +198,64 @@ function platform_main_feature_menu_default_menu_links() {
...
     'options' => array(
       'attributes' => array(
         'title' => '',
       ),
-      'identifier' => 'menu-header-menu_about-this-site:uuid/node/3dfbfeaf-cb77-4ac0-b16f-145af17649b5',
+      'identifier' => 'menu-header-menu_about-us:node/5',
     ),
     'module' => 'menu',
-    'hidden' => 1,
+    'hidden' => 0,
```
Comment (jordandukart) [inline platform_main_feature.features.menu_links.inc]: Not hidden anymore?

File: platform_main_feature.features.user_permission.inc [hunk 5]
```diff
@@ -21,49 +21,5 @@ function platform_main_feature_user_default_permissions() {
     'module' => 'google_analytics_reports',
   );
 
-  // Exported permission: 'export islandora bookmarks'.
```
Comment (jordandukart) [inline platform_main_feature.features.user_permission.inc:24]: Nuking permissions?

**Review by jordandukart — CHANGES_REQUESTED** [review]

File: platform_main_feature.features.user_permission.inc (hunk 5 above)
Comment (MorganDawe) [reply platform_main_feature.features.user_permission.inc:24]: Looks like the permissions are being handled in the bpl permissions feature

**Pull request merged**